modules/auxiliary/gather/aws_console.rb

```diff
@@ -0,0 +1,73 @@
+require 'msf/core'
+require 'metasploit/framework/aws/client'
```

Comment:
Minor, but I think the way this is usually done is by putting it in one of the common mixins.  Perhaps https://github.com/rapid7/metasploit-framework/blob/cac890a797d0d770260074dfe703eb5cfb63bd46/lib/msf/core/exploit/mixins.rb?